planing_event_bot.py:43 #INFO     [2025-03-12 08:31:46,561] - __main__ - Starting bot
base.py:507 #INFO     [2025-03-12 08:31:47,008] - apscheduler.scheduler - Adding job tentatively -- it will be properly scheduled when the scheduler starts
base.py:1090 #INFO     [2025-03-12 08:31:47,010] - apscheduler.scheduler - Added job "process_scheduler_send_task" to job store "default"
base.py:214 #INFO     [2025-03-12 08:31:47,010] - apscheduler.scheduler - Scheduler started
dispatcher.py:527 #INFO     [2025-03-12 08:31:47,373] - aiogram.dispatcher - Start polling
dispatcher.py:341 #INFO     [2025-03-12 08:31:47,464] - aiogram.dispatcher - Run polling for bot @Events_Planner_Bot_Bot id=7613252869 - 'EventPlanner'
start_handlers.py:39 #INFO     [2025-03-12 08:36:35,899] - root - process_start_command
requests.py:89 #INFO     [2025-03-12 08:36:35,900] - root - get_user_by_id 943040396
dispatcher.py:172 #INFO     [2025-03-12 08:36:36,314] - aiogram.event - Update id=824284751 is handled. Duration 422 ms by bot id=7613252869
dispatcher.py:554 #INFO     [2025-03-12 08:37:06,157] - aiogram.dispatcher - Polling stopped
dispatcher.py:219 #ERROR    [2025-03-12 08:37:06,278] - aiogram.dispatcher - Failed to fetch updates - TelegramNetworkError: HTTP Client says - ClientOSError: [Errno 1] [SSL: APPLICATION_DATA_AFTER_CLOSE_NOTIFY] application data after close notify (_ssl.c:2685)
dispatcher.py:221 #WARNING  [2025-03-12 08:37:06,278] - aiogram.dispatcher - Sleep for 1.000000 seconds and try again... (tryings = 0, bot id = 7613252869)
dispatcher.py:359 #INFO     [2025-03-12 08:37:06,430] - aiogram.dispatcher - Polling stopped for bot @Events_Planner_Bot_Bot id=7613252869 - 'EventPlanner'
